lcls_tools/common/measurements/emittance_measurement.py

```diff
@@ -85,20 +86,18 @@ def measure(self):
         kmod = bdes_to_kmod(self.energy, magnet_length, np.array(self.scan_values))
 
         # compute emittance and bmag
-        emittance, bmag, _, _ = compute_emit_bmag(
+        results = compute_emit_bmag(
             k=kmod,
             beamsize_squared=beamsize_squared,
             q_len=magnet_length,
             rmat=rmat,
             twiss_design=twiss_betas_alphas,
         )
 
-        results = {
-            "emittance": emittance,
-            "BMAG": bmag,
+        results = results.update({
```

Comment:
Do we need the `results =` here if we are updating?

Reply:
That actually breaks things, because update returns None.

Reply:
thanks for catching that